action.yaml

```diff
@@ -27,28 +27,41 @@ inputs:
   delete-branch:
     description: 'Delete PR branch after merge'
     required: false
-    default: true
+    default: 'true'
   branch-prefix:
     description: 'The prefix for the name of the feature branch'
     required: false
     default: 'automated-code-release-'
+  working-directory:
+    description: 'Directory to run semantic-release in '
+    required: false
```

Comment:
Uh... not required, but no default?

Reply:
I believe that no default is essentially null or undefined, so it's like the parameter that uses the input is never actually specified.